src/Markdig.Benchmarks/Markdig.Benchmarks.nuget.targets

```diff
@@ -3,11 +3,11 @@
   <PropertyGroup Condition=" '$(ExcludeRestorePackageImports)' != 'true' ">
     <RestoreSuccess Condition=" '$(RestoreSuccess)' == '' ">True</RestoreSuccess>
     <RestoreTool Condition=" '$(RestoreTool)' == '' ">NuGet</RestoreTool>
-    <ProjectAssetsFile Condition=" '$(ProjectAssetsFile)' == '' ">C:\Code\lunet\markdig\src\Markdig.Benchmarks\project.lock.json</ProjectAssetsFile>
+    <ProjectAssetsFile Condition=" '$(ProjectAssetsFile)' == '' ">C:\Users\Dave\Documents\Code Repository\markdig\src\Markdig.Benchmarks\project.lock.json</ProjectAssetsFile>
```

Comment:
This looks like an unnecessary (maybe dangerous?) change.
It makes me wonder whether this file should be versioned in git in the first place since it seems to be containing user-specific data. @xoofx what do you think?